src/main/java/com/easypost/http/Requestor.java

```diff
@@ -371,6 +371,7 @@ private static EasyPostResponse makeURLConnectionRequest(final RequestMethod met
                             String.format("Unrecognized HTTP method %s. Please contact EasyPost at %s.", method,
                                     Constants.EASYPOST_SUPPORT_EMAIL));
             }
+            conn.setRequestProperty("Accept", "*/*");
```

Comment:
issue: We should only be accepting JSON back from the API because we only know how to handle JSON in the lib. The other libs are set to JSON as well